src/typescript/util/mimeTypes.ts

```diff
@@ -0,0 +1,84 @@
+const mimeTypes: Record<string, string> = { //These are the most common MIME types according to MDN. Will add more intermittently.
```

Comment:
Pick the most common file types that normal people use, not just what MDN says. Some of these I've never heard of. Do common archives (zip, 7z, rar, tar, gz, tar.gz, etc) Documents (txt, csv, pdf, docx, doc, ppt, pptx, etc.) Videos (avi, mp4, mov, etc) Audio (mp3, wav, flac, aac, etc) Images (jpg, jpeg, bmp, svg, png, ico) Executables (exe, dmg, pkg, etc). I'll list the ones to remove below